products/vuetify.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vuetify
+category: framework
+iconSlug: vuetify
+permalink: /vuetify
+alternate_urls:
+-   /vuetify
+
+versionCommand: npm list vuetify
+releasePolicyLink: https://vuetifyjs.com/en/introduction/roadmap/#released
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+-   npm: vuetify
+
+releases:
+-   releaseCycle: "3"
+    support: true
+    eol: false
+    latest: "3.1.12"
+    lts: true
+    latestReleaseDate: 2023-03-28
+    releaseDate: 2022-11-01
+    link: https://github.com/vuetifyjs/vuetify/releases/tag/v3.1.12
+
+-   releaseCycle: "2"
+    support: true
+    eol: false
+    latest: "2.6.14"
+    lts: true
+    latestReleaseDate: 2023-01-10
+    releaseDate: 2019-07-23
+    link: https://github.com/vuetifyjs/vuetify/releases/tag/v2.6.14
+
+-   releaseCycle: "1.5"
+    support: 2020-07-31
+    eol: 2020-07-31
+    latest: "1.5.23"
+    lts: true
+    latestReleaseDate: 2020-02-21
+    releaseDate: 2019-02-05
+    link: https://github.com/vuetifyjs/vuetify/releases/tag/v1.5.23
+
+---
+
+> [Vuetify](https://vuetifyjs.com/) is a no design skills required UI Library with 
+> beautifully handcrafted Vue Components.
+
+[Vuetify release cycle](https://vuetifyjs.com/en/introduction/long-term-support/). 
+Vuetify provides long-term support to the last major release for 18 months
+for critical bugs and security vulnerabilities.
+
+We understand that many projects that utilize Vuetify have development cycles
+that prevent upgrading to the latest version. In order to provide developers
+and businesses peace of mind when adopting Vuetify, we commit to at minimu
+of 18 months for critical bugs and security vulnerabilities for the latest minor of the last major release.
```

Comment:
We use [a neutral third-person tone](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#tone-and-text), so avoid using "we". In this case, all of this is covered above already, so dropping this para.

```suggestion
```